source/agora/flash/Channel.d

```diff
@@ -727,10 +727,16 @@ public class Channel
         PrivateNonce priv_nonce = genPrivateNonce();
         PublicNonce pub_nonce = priv_nonce.getPublicNonce();
 
-        auto new_balance = this.foldHTLCs(this.cur_balance, secrets, rev_htlcs,
+        const old_balance = this.cur_balance;
```

Comment:
I think there's no need to introduce the new variable `old_balance`. Is there any reason for that?

Reply:
Yep, PRs in the near future will be using that.